inc/ticket.class.php

```diff
@@ -347,11 +347,21 @@ static public function displayVoucherInTicketProcessingForm($params) {
          $out .= __('Voucher name', 'credit');
          $out .= "</label>";
          $out .= "</td><td>";
+         $entity = $ticket->getEntityID();
+         $entities = getAncestorsOf('glpi_entities', $entity);
+         $entities[] = $entity;
          $out .= PluginCreditEntity::dropdown(['name'      => 'plugin_credit_entities_id',
-                                               'entity'    => $ticket->getEntityID(),
```

Comment:
You should not remove this line, and should just add `'entity_sons' => true` option.